src/components/Bookmark/Group/Header/header.module.scss

```diff
@@ -0,0 +1,36 @@
+.bookmark-group__header {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	height: 60px;
+	box-sizing: border-box;
+
+	h1 {
+		font-weight: 400;
+		font-size: 12pt;
```

Comment:
`pt` is not the best choice. `Points` depends on the screen and the resolution and is not recommended for using for screens. See https://www.w3.org/Style/Examples/007/units.en.html
I would suggest replacing it with `rem` if you want to make it relative or just `px`

Reply:
Ok, changed to `px` throughout the project